cps2pc

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+import argparse
+import cps
+from collections import OrderedDict
+import os
+import re
+import subprocess
+import sys
+
+_supported_compilers = ['gcc', 'clang']
+_default_prefix = '${pcfiledir}/../..'
+#------------------------------------------------------------------------------
+def get_link_features(link_features, platform):
+    _supported_kernels = ['cygwin', 'linux', 'darwin']
```

Comment:
I want to say these would make more sense as globals, all at the top, than scattered throughout the file? Is there a strong reason not to do that?

Reply:
I just tried to keep these as local as possible, but I will group them as globals.

Reply:
Understood. There's an argument to be made either way, but one reason I tend to define all this stuff up front is that it makes it easier to read and modify them. Given that was a stated goal in your commit message... :slightly_smiling_face: